editor/components/document-title/index.js

```diff
@@ -16,7 +16,7 @@ class DocumentTitle extends Component {
 	}
 
 	setDocumentTitle( title ) {
-		document.title = title + ' | ' + this.originalDocumentTitle;
+		document.title = 'Edit: ' + title + ' | ' + this.originalDocumentTitle;
```

Comment:
Should we use `sprintf` and `__` for i18n purpose?

Reply:
Specifically, we'd want to allow for this string to be translateable, and since we don't want to presume that it will be shown as a prefix in all languages, we'd want to use placeholders here.

https://codex.wordpress.org/I18n_for_WordPress_Developers#Placeholders

The above documentation is for the PHP functions, but parallels exist for JavaScript in this project:

https://github.com/WordPress/gutenberg/tree/master/i18n

You can bring these into scope by importing them at the top of the file:

```js
/**
 * WordPress dependencies
 */
import { sprintf, __ } from '@wordpress/i18n';
```

See: https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#javascript

Reply:
> since we don't want to presume that it will be shown as a prefix in all languages, we'd want to use placeholders here

I think we should assume it would be a prefix in all languages. The point is to make it so that “Edit” appears as the first word in the tab. If it is at the end then it could be confused with the post on the frontend. From looking at core, there seem to be quite a few examples of fixed strings appearing at the beginning without the ability to move them around.

Reply:
But does it _make sense_ (i.e. can it be read as such) as a prefix in all languages? I am reaching the extent of my knowledge of localization, so the answer could very well be "yes" for all I know.